src/protocols/secure_channel/CASESession.cpp

```diff
@@ -627,6 +627,7 @@ CHIP_ERROR CASESession::HandleSigma1(System::PacketBufferHandle && msg)
     {
         ChipLogError(SecureChannel, "CASE failed to match destination ID with local fabrics");
         ChipLogByteSpan(SecureChannel, destinationIdentifier);
+        ExitNow();
```

Comment:
Suggest `VerifyOrExit(err == CHIP_NO_ERROR, err)` or something more often seen than `ExitNow()`. Not a big deal. Would even rather have `goto exit` than `ExitNow()`.

Reply:
Switched to `SuccessOrExit(err);` outside of the if/else.